src/tools/__tests__/buildGoogleWebfontUrl.test.js

```diff
@@ -0,0 +1,42 @@
+import { buildGoogleWebfontUrl } from '../styled'
+
+describe('buildGoogleWebfontUrl', () => {
+
+  it('returns no URL if fontFamily is missing', () => {
+    const url = buildGoogleWebfontUrl(null, null)
+    expect(url).toEqual('')
+  })
+
+  describe('with fontFamily present', () => {
+    const baseUrl = 'https://fonts.googleapis.com/css2'
+
+    it('returns a correct URL if fontWeight=null AKA (intentially) missing', () => {
+      const url = buildGoogleWebfontUrl('Fira Sans', null)
+      expect(url).toEqual(`${baseUrl}?family=Fira+Sans`)
+    })
+
+    it('returns a correct URL with fontWeight=100 – a correct weight', () => {
+      const url = buildGoogleWebfontUrl('Fira Sans', 100)
+      expect(url).toEqual(`${baseUrl}?family=Fira+Sans:wght@100`)
+    })
+
+    it('returns a broken URL with fontWeight=33 – an incorrect weight', () => {
+      const url = buildGoogleWebfontUrl('Fira Sans', 33)
+      expect(url).toEqual(`${baseUrl}?family=Fira+Sans:wght@33`)
+      // We don't catch this case since it will clearly fail (font does not show),
+      // and we could only validate the weight by checking with the GoogleWebfonts API first.
+    })
+
```

Comment:
Should test those, too.
```suggestion

    it('returns a correct URL with fontWeight="light"', () => {
      const url = buildGoogleWebfontUrl('Fira Sans', "Some String")
      expect(url).toEqual(`${baseUrl}?family=Fira+Sans:wght@300`)
    })

    it('returns a correct URL with fontWeight="bold"', () => {
      const url = buildGoogleWebfontUrl('Fira Sans', "Some String")
      expect(url).toEqual(`${baseUrl}?family=Fira+Sans:wght@700`)
    })
```